Makefile

```diff
@@ -98,7 +98,10 @@ clean::
 	rm -f *.gcov
 
 .PHONY: test
-test:: lcov functest
+test:: test_docker_image lcov functest
+
+test_docker_image:
```

Comment:
We don't actually need this image anymore
It is a part of the old tests that is no longer used

Reply:
Ok. I'll roll this back in a follow-up PR later today - I need to make a few small corrections .. change location of map used for EDT to its own header file instead of using shared_map_defs.h as this is not shared with any of the other XDP code.

Should I get rid of the test target as well from the Makefile to avoid confusion? (We keep run_unittests)